oriedita/src/main/java/oriedita/editor/swing/dialog/ExportDialog.java

```diff
@@ -170,7 +170,7 @@ private void onClose() {
         final JPanel panel3 = new JPanel();
         panel3.setLayout(new GridLayoutManager(1, 2, new Insets(0, 0, 0, 0), -1, -1));
         contentPane.add(panel3, new GridConstraints(0, 0, 1, 1, GridConstraints.ANCHOR_CENTER, GridConstraints.FILL_BOTH, GridConstraints.SIZEPOLICY_CAN_SHRINK | GridConstraints.SIZEPOLICY_CAN_GROW, GridConstraints.SIZEPOLICY_CAN_SHRINK | GridConstraints.SIZEPOLICY_CAN_GROW, null, null, null, 0, false));
-        list1 = new JList();
+        list1 = new JList<>();
```

Comment:
This is generated code btw, so i'm not sure if these changes will persist.

Reply:
Huh. Well I've tried.